recipes/opencv/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set version = "2.4.12" %}
+{% set filename = "opencv2-%s.zip" % version %}
+
+package:
+  name: opencv
+  version: {{ version }}
+
+source:
+  fn: {{ filename }}
+  url: https://github.com/Itseez/opencv/archive/{{ version }}.zip
+  sha1: 2225768a48dbf99426bf77451a433ed432f54d8e
+
+  patches:
+   - osx_rpath.patch  # [osx]
+
+build:
+  number: 0
+  skip: true  # [not py27]
+
+requirements:
+  build:
+   - cmake
+   - curl
+   - numpy x.x
+   - python
+   - eigen 3.*
+   - zlib  # [unix]
+   - jpeg  # [unix]
+   - libtiff  # [unix]
+   - libpng  # [unix]
+
+  run:
+   - python
+   - numpy x.x
+   - zlib  # [unix]
+   - jpeg  # [unix]
+   - libtiff  # [unix]
+   - libpng  # [unix]
+
+test:
+  imports:
+    - cv2
+    - cv2.cv
+  commands:
+    # Verify dynamic libraries.
+    {% set opencv_libs = [
+            "core",
+            "flann",
+            "imgproc",
+            "highgui",
+            "features2d",
+            "calib3d",
+            "ml",
+            "video",
+            "legacy",
+            "objdetect",
+            "photo",
+            "gpu",
+            "nonfree",
+            "contrib",
+            "stitching",
+            "superres",
+            "videostab"
+    ] %}
+    {% for each_opencv_lib in opencv_libs %}
+    - test -f $PREFIX/lib/libopencv_{{ each_opencv_lib }}.dylib   # [osx]
+    - test -f $PREFIX/lib/libopencv_{{ each_opencv_lib }}.so      # [linux]
+    {% endfor %}
```

Comment:
As another test, can we verify that `FindOpenCV.cmake` got installed correctly? Ran into an issue with the exist OpenCV install where this was not the case. Would be nice if we have this out of the gate with this one?